component/attr_cache/attr_cache.go

```diff
@@ -153,8 +153,8 @@ func (ac *AttrCache) Configure(_ bool) error {
 
 	ac.noSymlinks = conf.NoSymlinks
 
-	log.Info("AttrCache::Configure : cache-timeout %d, symlink %t, cache-on-list %t",
-		ac.cacheTimeout, ac.noSymlinks, ac.cacheOnList)
```

Comment:
This max files is a hard coded value if i remember correctly. Is there a way user can configure this ?

Reply:
It is configurable as seen in baseConfig